src/server/graphql/ScreenshotDiffType.js

```diff
@@ -0,0 +1,81 @@
+import {
+  GraphQLObjectType,
+  GraphQLString,
+  GraphQLID,
+  GraphQLEnumType,
+} from 'graphql';
+import ScreenshotDiff from 'server/models/ScreenshotDiff';
+
+export const resolve = (source, args) => {
+  return ScreenshotDiff
+    .query()
+    .where({
+      build_id: args.id,
+    })
+    .then((a) => {
```

Comment:
It's a WIP, I have submitted the PR before taking the train. I need filter with the build id.